skimage/morphology/misc.py

```diff
@@ -7,13 +7,13 @@
 
 # Our function names don't exactly correspond to ndimages.
 # This dictionary translates from our names to scipy's.
-funcs = ('erosion', 'dilation', 'opening', 'closing')
-skimage2ndimage = {x: 'grey_' + x for x in funcs}
+funcs_gray = ('erosion', 'dilation', 'opening', 'closing')
+skimage2ndimage = {x: 'grey_' + x for x in funcs_gray}
 
 # These function names are the same in ndimage.
-funcs = ('binary_erosion', 'binary_dilation', 'binary_opening',
-         'binary_closing', 'black_tophat', 'white_tophat')
-skimage2ndimage.update({x: x for x in funcs})
+funcs_binary = ('binary_erosion', 'binary_dilation', 'binary_opening',
+                'binary_closing', 'black_tophat', 'white_tophat')
+skimage2ndimage.update({x: x for x in funcs_binary})
```

Comment:
I love this change. =)

Reply:
Otherwise mypy was complaining that the length had changed...